### DISCUSSION TITLE
HTTPS enabled on port 4000

### DISCUSSION THREAD

**Discussion by aacostarua** [blockscout/blockscout#9147] [Q&A]
Right now I have my blockscout client running normally on HTTP 4000, but for some reason it can't be reached when I try connecting 4000 HTTPS.  Previously blockscout was on the main route but now I would like it just on 4000 HTTPS. I still have the /etc/hosts changes as well that were provided in the blockscout docs for HTTPS. Please let me know how I should fix this issue.

**Comment by GimluCom** [comment]
Are you using the new frontend or the old interface?
If you want to run the backend over https, you may want to consider using a reverse proxy (nginx). The docker-compose version of blockscout is already configured with nginx.

**Reply by aacostarua** [reply]
Its newer. I'm trying to run it manually I just get SSL_ERROR_RX_RECORD_TOO_LONG. I don't think reverse proxy Nginx will help as the app doesn't except SSL on 4000.

**Reply by GimluCom** [reply]
SSL would be configured on Nginx. Nginx would still connect to blockscout locally over http without SSL.

**Reply by aacostarua** [reply]
I tried Nginx reverse proxy way earlier and not everything loaded from the 4000 port to /explorer with the HTTPS redirect.  Is there a way to direct mix / elixir to the lets encrypt certs I have working, to automatically except HTTPS on the app Blockscout level.  I was able to achieve this with my other backend and frontend apps.

**Comment by ekawibisono** [comment]
hi @aacostarua any solved this problem ?

**Reply by aacostarua** [reply]
Nope.

**Reply by ekawibisono** [reply]
you can join discord [blockscout](https://discord.com/invite/blockscout) to continue your problem. @aacostarua
